elements/table.js

```diff
@@ -196,12 +194,15 @@ function tableElement (dats, send) {
 // ([obj], fn) -> html
 function createTable (dats, send) {
   return dats.map(dat => {
-    const progress = dat.stats.blocksTotal
-      ? Math.round(dat.stats.blocksProgress / dat.stats.blocksTotal * 100)
+    const stats = dat.stats && dat.stats.get()
```

Comment:
shouldn't `dat.stats` always be set? also, this getter was more convenient to use: https://github.com/datproject/dat-desktop/pull/198/files#diff-66caf64acff533dbfc525b13489309cdL178